src/disco/components/DiscoPane/styles.scss

```diff
@@ -16,8 +16,9 @@
 
 .disco-content {
   color: $header-copy-font-color;
-  font-size: 14px;
-  line-height: 1.28;
+  font-size: 16px;
```

Comment:
@willdurand could this use a var?

Reply:
I think so yes, next iteration on the disco pane will bring better SCSS.